recipes/mmult/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "MMULT" %}
+{% set version = "0.0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/lijinbio/MMULT/archive/v{{ version }}.tar.gz
+  sha256: d39aeeccd3f0aa5851a9db38369ae7beb8ffc0389f65340fc786a4dcff2f49b7
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - boost
+    - sundials
+    - rapidjson
+    - eigen
+    - tbb-devel
+  host:
+    - boost
+    - sundials
+    - rapidjson
+    - eigen
+    - tbb-devel
+    - r-base >=4
+    - r-ggplot2
+    - r-gridextra
+  run:
+    - boost
```

Comment:
boost can be removed here I believe

Reply:
@bgruening Unfortunately no. Removing boost (or boost-cpp) will cause broken dependent boost libraries because the binary is dynamically linking boost libraries. Therefore, boost-cpp is used to replace boost here in the run section. Thank you for all your comments. My revision has passed the CI build. Can you help review my PR again? Thank you.